index.html

```diff
@@ -53,95 +56,108 @@ <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 3 -->
 
         <!-- Start of Project 4 -->
         <div id="tonic-project">
             <div class="data">
-                <img src="./images/snapshots/Snapshoot Portfolio 4.png" alt="Tonic project snapshot">
-                <h2 class="project-title">Multi-Post <br />Stories</h2>
-                <div class="work-nature">
-                    <label class="canopy">CANOPY</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="backend"> Back End Dev</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="year">2015</label>
+                <div class="image-div"><img src="./images/snapshots/Snapshoot Portfolio 4.png" alt="Tonic project snapshot"></div>
+                <div class="desktop-image-div"><img class="desktop-snapshop" src="./images/desktop/Snapshoot-Desktop4.png" alt="Tonic project snapshot"></div>
+                <div class="project-content-4">
+                    <h2 class="project-title">Multi-Post <br />Stories</h2>
+                    <div class="work-nature">
+                        <label class="canopy">CANOPY</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="backend"> Back End Dev</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="year">2015</label>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tools">
+                        <li><label class="tool-press">HTML</label></li>
+                        <li><label class="tool-press">CSS</label></li>
+                        <li><label class="tool-press">JAVASCRIPT</label></li>
+                    </ul>
+                    <button class="see-project">See Project</button>
                 </div>
-                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="tools">
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 4 -->
     </div>
+
     <section id="skills-page">
         <div class="about-description">
```

Comment:
- [ ] Kindly remove the spacing between the contact section and about.
### Your design
![image](https://user-images.githubusercontent.com/94551484/229281569-2678f09f-d50b-42d3-b914-14c854e0cca9.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/229281684-dbc423d6-89d4-4d6e-a9c4-71a8428b676f.png)
- [ ] Kindly fix the button of the contact section to match the design template and add space below it.
### Figma
![image](https://user-images.githubusercontent.com/94551484/229281959-a4ee527f-6357-475e-b8a2-39d31a6710c8.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/229282046-bffb2969-f8a5-41af-948b-96e6b06fb307.png)